File: pep/level1/basics/gettingstarted/PrimeSeries.java
import java.util.Scanner;
import java.lang.Math;
public class PrimeSeries {
  public static void main(String [] args) {

    int low = 0, high = 0;
    Scanner s = new Scanner(System.in);
    System.out.println("Enter lower range:");

    if(s.hasNext())
      low = s.nextInt();
    System.out.println("Enter higher range:");
    if(s.hasNext())
      high = s.nextInt();

    for(int num = low; num <= high; num++){
        int div = 2;
        while(div * div <= num){
            if(num % div == 0){
                break;
            }
            div++;
        }

        if(div * div > num){
            System.out.println(num);
        }
    }
  }
}

File: pep/level1/basics/gettingstarted/PrimeSeries.cpp
#include <iostream>
using namespace std;

int main() {

  int low, high;

      cin>>low;
      cin>>high;

      int div = 2;
      while(low <= high) {
        int div = 2;
        while((div*div)<low) {
        if(low%div == 0) {
           break;
        }
        div++;
      }

      if((div * div) > low)
      cout << low << endl;
      low++;
    }


  return 0;
}
